src/main/java/List.java

```diff
@@ -0,0 +1,165 @@
+import java.util.Scanner;
+
+public class List {
+    ArrayList<Task> tasks =new ArrayList<>();
+    private static int size;
+
+    public List() {
+        this.size = 0;
+    }
+
+    public void printList() {
+        for (int i=0;i<tasks.size();i++) {
+            System.out.println(i+1+". " +tasks.get(i).toString());
+        }
+        System.out.println("");
+    }
+
+
+    public void addDeadline (String description, String by) {
+        tasks.add(new Deadline(description,by));
+        System.out.println("Got it. I've added this task:");
+        System.out.println(tasks.get(this.size).toString());
+        this.size++;
+        System.out.println("Now you have " + this.size + " tasks in the list.\n");
+    }
+
+    public void addOldDeadline (String description, String by, Boolean isDone) {
+        tasks.add(new Deadline(description,by,isDone));
+        this.size++;
+    }
+
+
+    public void addEvent(String description, String at) {
+        tasks.add(new Event(description, at));
+        System.out.println("Got it. I've added this task:");
+        System.out.println(tasks.get(this.size).toString());
+        this.size++;
+        System.out.println("Now you have " + this.size + " tasks in the list.\n");
+    }
+
+    public void addOldEvent (String description, String at, Boolean isDone) {
+        tasks.add(new Event(description,at, isDone));
+        this.size++;
+    }
+
+    public void addTodo (String description) {
+        tasks.add(new Todo(description));
+        System.out.println("Got it. I've added this task:");
+        System.out.println(tasks.get(this.size).toString());
+        this.size++;
+        System.out.println("Now you have " + this.size + " tasks in the list.\n");
+    }
+
+    public void addOldTodo (String description) {
+        tasks.add(new Todo(description));
+        this.size++;
+    }
+
+
+    public void isComplete(String command) {
+        command = command.replace("done", " ");
+        command = command.strip(); //removes white space
+        int index;
+        index = Integer.parseInt(command);
+        index--;
+        tasks.get(index).markTaskAsDone();
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println(tasks.get(index).toString() + "\n");
+    }
+
+    public void toRemove (String taskRemove) {
+        int index;
+        index= Integer.parseInt(taskRemove)-1;
+        System.out.println("Noted. I've removed this task:");
+        System.out.println(tasks.get(index).toString());
+        tasks.remove(index);
+        this.size--;
+        System.out.println("Now you have "+ this.size+ " tasks in the list.\n");
+    }
+
+
+
+
+    private static String PATH = new File("").getAbsolutePath();
+    static File filePath = new File(PATH + "/duke.txt");
+
+    public void readOldTextFile () {
+        //1. read old task list
+        //2. split into T/E/D, done and description and at/by
+        //3. add as new task into tasks arraylist
```

Comment:
You may want to comment above the relevant section of the code for clarity. Even better, you may want to think of how you can write the code so that it is clear even without these comments.